languages/go/exercises/concept/comments/.docs/after.md

```diff
@@ -0,0 +1,69 @@
+In the previous exercise, we saw that there are two ways to write comments in Go: single line comments that are preceded by `//`, and multiline comment blocks that are wrapped with `/*` and `*/`.
+
+## Comments for documentation
+
+In Go, comments play an important role in documenting code. They are used by the tool [godoc][godoc], which extracts these comments to create documentation about Go packages. A documentation comment should be a complete sentence that starts with the name of the thing being described and ends with a period.
+
+For comments other than documentation comments, while they may be helpful for maintainers, a [good rule of thumb][less comments] and more sustainable solution is to write code that is easier to understand so that explanatory comments are hopefully not needed.
+
+## Comments for exported identifiers
+
+Documentation comments should precede packages as well as [exported identifier][exported identifiers]s, e.g. exported functions, methods, package variables, constants, and structs, which you will learn more about in the next exercises. Comments written for packages and exported identifiers are useful for the users who import and use these packages.
+
+Note, however, that identifiers (such as variables) that are declared inside of functions and methods are private and do not necessarily require comments for the users of the packages.
+
+A package variable can look like this:
+
+```go
+// TemperatureFahrenheit gives a certain
+// temperature in degrees Fahrenheit.
+var TemperatureFahrenheit int
+```
+
+Note that `TemperatureFahrenheit` is capitalized, which makes this exported identifier a global variable.
+
+## Package comments
+
+Package comments should be written directly before a package and begin with `Pacakge X ...` like this:
+
+```go
+// Package kelvin provides tools to convert
+// temperatures to and from Kelvin.
+package kelvin
+```
+
+## Function comments
+
+A function comment can look like this:
```

Comment:
```suggestion
A function comment should be written directly before the function declaration, and takes the form `Function X ...` (a full sentence explaining what arguments the function takes, what it does with them, and what its return values mean, ending in a period):
```

Reply:
The function comment has to start with the function name, not "Function X ...". `golint` will say this if you try that:

```
 ~/repos/TestMain > golint .
main.go:12:1: comment on exported function Test should be of the form "Test ..."````

Reply:
Hey @tehsphinx, thank you for your feedback. This [new commit](https://github.com/exercism/v3/pull/1663/commits/d68af0ed27989e4758fcf06733fd96767420e3ee) fixes this!

Reply:
Thanks for spotting that what I obviously meant to say was "takes the form 'X ...'" 😁